index.yaml

```diff
@@ -4320,6 +4320,67 @@ entries:
     - https://github.com/newrelic/helm-charts/releases/download/newrelic-pixie-0.1.0-alpha.1/newrelic-pixie-0.1.0-alpha.1.tgz
     version: 0.1.0-alpha.1
   nri-bundle:
+  - apiVersion: v1
```

Comment:
Mostly out of curiosity, I wonder why its apiVersion 1 here. But it does not seem to be a problem.

Reply:
I believe the nri-bundle is V1, nri-kubernetes-v3 is V2